js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js

```diff
@@ -397,6 +411,11 @@
 				return;
 			}
 
+			if (keycode === 46 || keycode === 8) {
```

Comment:
We don't have a convention for this, but I think this would be more clear. If you have a better convention for those variable names please use that.

```javascript
var isDeleteKeyIsPressed = keycode === 46;
var isBackspaceKeyIsPressed = keycode === 8;

if (isDeleteKeyIsPressed || isBackspaceKeyIsPressed) {
```